src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -111,19 +115,21 @@ const ProteinsListItemInfo = (props: Props) => {
     LoadingState.LOADING
   );
 
-  const [summaryStatsLoadingState, setSummaryStatsLoadingState] = useState<
-    LoadingState
-  >(LoadingState.LOADING);
+  const [
+    summaryStatsLoadingState,
+    setSummaryStatsLoadingState
+  ] = useState<LoadingState>(LoadingState.LOADING);
 
   const proteinId =
     transcript.product_generating_contexts[0].product.unversioned_stable_id;
 
   const { product: productWithProteinDomains } =
     transcriptWithProteinDomains?.product_generating_contexts[0] || {};
 
-  const uniprotXref = transcript.product_generating_contexts[0].product.external_references.find(
-    (xref) => xref.source.id === SWISSPROT_SOURCE
+  const proteinXrefs = getProteinXrefs(
+    transcript.product_generating_contexts[0].product.external_references
```

Comment:
It might be better if we just pass `transcript` to the function `getProteinXrefs` and let it handle the rest.

Reply:
thanks @msmanoj so I export Transcript type from this component and use as the function props?

Reply:
changes made :-)